akka-projection-grpc/src/main/scala/akka/projection/grpc/replication/javadsl/ReplicationSettings.scala

```diff
@@ -87,7 +87,7 @@ object ReplicationSettings {
       .getConfigList("replicas")
       .asScala
       .toSet
-      .map { config: Config =>
+      .map { (config: Config) =>
```

Comment:
@patriknw Since you [asked](https://github.com/sebastian-alfers/akka-projection/pull/1#discussion_r1111576163) before, yes this is need. Otherwise there is a
```
ReplicationSettings.scala:90:15: missing parameter type
[error]       .map { (config) =>
```